src/components/step-sequence/step-sequence.component.tsx

```diff
@@ -20,15 +20,16 @@ export const StepSequence = ({
   ...props
 }: StepSequenceProps) => {
   return (
-    <StepSequenceStyle
+    <StyledStepSequence
       data-component="step-sequence"
       orientation={orientation}
+      p={0}
```

Comment:
I am assuming we want to allow consumers to control padding now as well as margins - so should we update `StepSequenceProps` to extend `SpaceProps` instead of `MarginProps`?